XboxKeyboardMouse/XboxController/Emulation/Activate.cs

```diff
@@ -76,38 +74,37 @@ class Activate {
             Program.MainForm.StatusWaiting();
         }
 
-        public static void SendtoController(X360Controller controller) {
-            byte[] report = controller.GetReport();
-            byte[] output = new byte[8];
-
-            // Send our data back to the virtual scp bus.
-            scpbus.Report(CONTROLLER_NUMBER, report, output);
+        public static void ShutDown()
+        {
+            if (shuttingDown == false)
```

Comment:
Is this if statement needed?
Code is difficult to read.  Why would we call ShutDown() and need to check !shuttingDown?

Reply:
Good point.  We need to track shuttingDown but can ditch this conditional and just set it; simPad.ShutDown is safe to call redundantly.